e2e/testdata/root-sync-helm-chart-cr.yaml

```diff
@@ -18,6 +18,11 @@ metadata:
   name: root-sync
   namespace: config-management-system
 spec:
+  override:
```

Comment:
Add a comment explaining why this needs to be set and that it's fixed in 1.17

Reply:
Done.